Makefile

```diff
@@ -1,4 +1,4 @@
-OPENAPI_GENERATOR_VERSION ?= v5.4.0
+OPENAPI_GENERATOR_VERSION ?= v6.4.0
```

Comment:
is this changed on the golang sdk too?

Reply:
we should probably keep those in sync

Reply:
Yep! Hopefully as best we can. I think it's just adhoc right now.